bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/extensions/PersistenceExtensions.java

```diff
@@ -420,16 +421,18 @@ private static void internalPersist(Item item, TimeSeries timeSeries, @Nullable
                 if (!skipEqual) {
                     HistoricItem historicItem = itemIterator.next();
                     if (!forward && !historicItem.getState().equals(state)) {
-                        // Past stored value different from current value, so it must have updated since last persist.
-                        return ZonedDateTime.now();
+                        // Last persisted state value different from current state value, so it must have updated since
+                        // last persist. We do not know when.
```

Comment:
@J-N-K, @mherwege, do you think we should log a warning here, and also below?

Reply:
All warn logging in the persistence extensions point to a configuration issue. This is not a configuration issue. I don't think we should polute the log with warning messages for this. The method javadoc is clear about it.